runtime/doc/api.txt

```diff
@@ -1970,7 +1970,10 @@ nvim_clear_autocmds({opts})                            *nvim_clear_autocmds()*
                       `test.py` will not match the pattern.
                   • defaults to clearing all patterns.
                   • NOTE: Cannot be used with {buffer}
-                • buffer: (bufnr)
+                • buf: (bufnr)
+                  • clear only |autocmd-buflocal| autocommands.
+                  • NOTE: Cannot be used with {pattern}
+                • buffer: (bufnr) (deprecatd) use buf.
```

Comment:
IMO keeping the old form documented everywhere is just noise. (Especially as its repeating the same info as `buf`)